.github/workflows/integration.yaml

```diff
@@ -27,6 +27,17 @@ jobs:
           - 9091:9091
           - 9092:9092
 
+      kafka_confluent:
+        image: confluentinc/confluent-local:latest
```

Comment:
note to myself: Ideally we should always use tags (or better) SHAs. But this is not done in the other workflows either yet, making troubleshooting harder.

Reply:
Done with `latest` -> `7.6.0`.